cmd/tailscale/cli/debug.go

```diff
@@ -87,6 +90,16 @@ func runDebug(ctx context.Context, args []string) error {
 		os.Stdout.Write(goroutines)
 		return nil
 	}
+	if debugArgs.derpMap {
+		dm, err := prodDERPMap(ctx, http.DefaultClient)
```

Comment:
for this command, I don't think we should silently switch between the tailscaled derp map and the server's one.

Let's only use client/tailscale.CurrentDERPMap here. On error, it can log.Fatal some message with the URL to `ipn.DefaultControlURL/derpmap/default` or something.